.github/workflows/main.yml

```diff
@@ -25,11 +25,7 @@ jobs:
       - name: Install python packages
         run: |
           pip install joblib>=0.14.0 scikit-learn>=0.23.1 pytest pylint
```

Comment:
Can we update the testing matrix, adding joblib versions,
also ,we can reduce python versions for reducing testing CI jobs.


      matrix:
        JOBLIB_VERSION: ["1.2", "1.3"]
        PYTHON_VERSION: ["3.7"]
        PIN_MODE: [false, true]
        PYSPARK_VERSION: ["3.0.3", "3.1.3", "3.2.3", "3.3.2", "3.4.0"]